### PR TITLE
PLANET-7268 Add meta and opengraph html tags on listing pages

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-7268](https://jira.greenpeace.org/browse/PLANET-7268)

**Requirements:**
- All listing pages
  - [x] Add `og:site_name` and `og:url`. Use posts as a guide on how to populate those.
  - [x] Use "website" as `og:type`.
- Categories, Tags, Post Types, Action Types
  - [x] Use title and description from their edit page as meta title and description tags, but also as `og:title` and `og:description`.
- Author page
  - [x] Use "profile" as `og:type`.
- News & Stories
  - [x] Use assigned page title as `og:title` and meta title tag.
  - [x] Use assigned page excerpt as `og:description` and meta description tag.
  - [x] Use assigned page featured image as `og:image`.

### Testing

You can inspect pages to make sure the relevant tags are present, you can also use https://opengraph.xyz/ for validation.
- [Tag listing page](https://www-dev.greenpeace.org/test-rhea/tag/renewables/)
- [Post type listing page](https://www-dev.greenpeace.org/test-rhea/press/)
- [Category listing page](https://www-dev.greenpeace.org/test-rhea/category/energy/)
- [Action type listing page](https://www-dev.greenpeace.org/test-rhea/petitions/)
- [Author page](https://www-dev.greenpeace.org/test-rhea/author/tgarcia/)
- [News & stories page](https://www-dev.greenpeace.org/test-rhea/news-stories/)

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [rhea](https://www-dev.greenpeace.org/test-rhea) | [admin](https://www-dev.greenpeace.org/test-rhea/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-rhea/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-rhea) | [composer-local.json](https://github.com/greenpeace/planet4-test-rhea/blob/main/composer-local.json)

:watch: 2023.10.12 08:07:59

File: templates/blocks/meta_fields.twig [hunk 1]
```diff
@@ -25,10 +32,16 @@
...
     <meta property="og:url" content="{{ current_url }}" />
+    {% if (og_type) %}
+        <meta property="og:type" content="{{ og_type }}" />
+    {% else %}
+        <meta property="og:type" content="article" />
+    {% endif %}
     {% if (og_description) %}
         <meta property="og:description" content="{{ og_description|striptags }}" />
+    {% elseif (tag_description) %}
+        <meta property="og:description" content="{{ tag_description|striptags }}" />
```
[the remark this answers is not in the document]
Comment (mleray) [reply templates/blocks/meta_fields.twig:44]: We already have [this meta tag](https://github.com/greenpeace/planet4-master-theme/blob/da0714f221aba68699fb01bec1bff3b9423647b6/templates/blocks/meta_fields.twig#L23) for the description 🤔

[the remark this answers is not in the document]
Comment (lithrel) [reply templates/blocks/meta_fields.twig:44]: Haaa good point sorry :pray:

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**